src/app/testing/testing.helpers.ts

```diff
@@ -79,6 +79,7 @@ export const setupTestModuleForService = <T>(
   TestBed.configureTestingModule({
     imports: [TestingModule, ...(configuration?.imports ?? [])],
     providers: configuration?.providers ?? [],
+    teardown: { destroyAfterEach: false },
```

Comment:
Not sure what this does but it seems fairly broadly scoped - it won't break anything?

Reply:
Deleted. This snuck in a part of trying to get the tests to prove a live httpClient request would work. This is not required for the committed code.